media-sound/jack/files/shebang-python27.patch

```diff
@@ -0,0 +1,8 @@
+--- a/jack	2016-11-16 00:34:18.402706654 +0100
++++ b/jack	2016-11-16 00:34:09.994636061 +0100
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python
++#!/usr/bin/python2.7
```

Comment:
also, you're making it worse, the proper shebang should be
`#!/usr/bin/env python2`